.github/workflows/make-check.yml

```diff
@@ -13,6 +13,7 @@ jobs:
       matrix:
         os:
           - macOS-11
+          - macOS-12
```

Comment:
there's also `macOS-latest` if we don't care too much about the macOS version which is used.
macos-installer.yml also needs to be updated.

Reply:
@cfergeau `macOS-latest` still point to `macOS-11` :(

Reply:
It's being moved to macOS-12
See this post from October 3, 2022 https://github.blog/changelog/2022-10-03-github-actions-jobs-running-on-macos-latest-are-now-running-on-macos-12/

> Over the next 10 weeks, jobs using the macos-latest runner label will migrate from macOS 11 to 12.

Reply:
>  `macos-installer.yml` also needs to be updated.

This is still missing

Reply:
updated